Server/JokesOnYou.Web.Api/Models/LoginModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JokesOnYou.Web.Api.Models
+{
+    public class LoginModel
+    {
+        public string UserName { get; set; }
```

Comment:
Need some validation. I would recommend fluentvalidation but you can also use data annotations such as [Required]